src/DependencyInjection/SonataMediaExtension.php

```diff
@@ -63,6 +64,11 @@ public function load(array $configs, ContainerBuilder $container)
 
             if ('doctrine_orm' === $config['db_driver']) {
                 $loader->load('api_controllers.xml');
+                if (class_exists(Operation::class)) {
```

Comment:
It would be nice to have some comments about what to do when dropping `nelmio/api-doc-bundle` 2.x

Reply:
Agree. IMO this PRs are so big. I don't wanna force you to repeat check the same code. We should merge all this kind PRs to allow use api-doc v3 in sandbox. Then I will fix models for swagger and refactor it.

Reply:
We should not hurry anything. A PR should only be merged when there is nothing to change/fix.
These branch can be released at anything. And if some comment are needed, it's always better to have them in the same PR. Doing another way is the best way to create technical debt.

Reply:
We also can not talk about technical debt when we provide information about TODO in [dedicated issue](https://github.com/sonata-project/dev-kit/issues/778). In this case better for reviews it keep it in smaller RPs.

Reply:
The todo issue does not solve everything.
It's not because there is a todo that we can be sure it will done.
A release branch should never be in a `todo` state.

It's already hard enough for us to fix every `NEXT_MAJOR` comments.
It would be even harder without these comments.

I don't think adding comments to explain what to do when we'll drop the version will make this PR so much bigger.

Reply:
IMO it's always better to have this information in the code. When the time comes (you never know), there is no need to know that there is an issue somewhere. 

Even having the issue, with just a quick search in this case of `nelmio/api-doc-bundle` in the code, will be way easier and quicker to solve it.